pontoon/base/templatetags/helpers.py

```diff
@@ -250,3 +251,56 @@ def dict_html_attrs(dict_obj):
     return jinja2.Markup(' '.join(
         [u'data-{}="{}"'.format(k, v) for k, v in dict_obj.items()]
     ))
+
+
+def _serialize_elements(elements):
+    """Serialize text elements and references into a simple string."""
+    response = ''
+
+    for element in elements:
+        if type(element) == ast.TextElement:
+            response += element.value
+
+        elif type(element) == ast.ExternalArgument:
+            response += '{$' + element.id.name + '}'
```

Comment:
we usually try to put spaces around the placeable when we serialize: `{ $name }` rather than `{$name}` as in https://github.com/projectfluent/python-fluent/blob/master/fluent/syntax/serializer.py#L137